stdlib/promise.rb

```diff
@@ -146,7 +146,7 @@ def exception?
   end
 
   def realized?
-    !!@realized
+    @realized != false
```

Comment:
This is not equivalent, did you check if the new code is safe?

Reply:
Yes, `@realized` can be `false` or a symbol, also all specs pass (there are some in `spec/opal/stdlib/promise/*.rb` I guess)